source/includes/_outliers.md

```diff
@@ -3,9 +3,15 @@
 curl https://api.gresb.com/api/v1/entities/{entity_id}/assets \
   -H "Authorization: Bearer $ACCESS_TOKEN"
 ```
-GRESB provides a realtime outlier check to ensure data quality and automatically flags
-values that seem to be out of the norm.Outlier checks are performed after creating or updating a Portfolio asset, 
-errors occurred. If no validation outliers have been detected, the `outliers:[]` field in the response will list them accordingly.
+GRESB provides a realtime outlier check to ensure data quality, and automatically flags
+values that seem to be out of the norm. Outlier checks are performed after creating or updating a portfolio asset.
+
+If no validation outliers have been detected, the `outliers:[]` field in the response will list them accordingly.
+
+In the case `status` field reads `true`, it means a soft outlier had been flagged:
+The data points associated to this outlier will be included in your score, but not in the calculation of the benchmark groups.
```

Comment:
we're missing `false` right?

I also don't get where `status` is coming from?

<img width="399" alt="image" src="https://github.com/GRESB/api-docs/assets/29230216/455ddb69-52d8-4f5d-857b-1e0bb098d084">

```
            "kpi": "en",
            "type": "lfl",
            "accepted": true,
            "value": "-28.20623403"
```

Reply:
I think I meant accepted instead of status, thank you for the catch.

And with your screenshot now I came to notice that what I communicated is already communicated, my edit causing duplication. I will be removing my addition about the soft/hard outliers.